Sources/LoggerAPI/Logger.swift

```diff
@@ -207,23 +253,77 @@ public class Log {
         lineNum: Int = #line, fileName: String = #file) {
             if let logger = logger, logger.isLogging(.exit) {
                 logger.log(.exit, msg: msg(),
-                    functionName: functionName, lineNum: lineNum, fileName: fileName)
+                           functionName: functionName, lineNum: lineNum, fileName: fileName)
             }
+            swiftLogger?.trace("\(msg())")
     }
     
-    /// Indicates if a message with a specified type (`LoggerMessageType`) will be in the logger
-    /// output (i.e. will not be filtered out).
+    /// Indicates if a message with a specified type (`LoggerMessageType`) will be logged
+    /// by some configured logger (i.e. will not be filtered out). This could be a Logger
+    /// conforming to LoggerAPI, swift-log or both.
     ///
-    /// - Parameter type: The type of message (`LoggerMessageType`).
+    /// - Parameter level: The type of message (`LoggerMessageType`).
     ///
     /// - Returns: A Boolean indicating whether a message of the specified type
-    ///           (`LoggerMessageType`) will be in the logger output.
+    ///           (`LoggerMessageType`) will be logged.
     public class func isLogging(_ level: LoggerMessageType) -> Bool {
+        return isLoggingToLoggerAPI(level) || isLoggingToSwiftLog(level)
+    }
+
+    /// Indicates whether a LoggerAPI Logger is configured to log at the specified level.
+    ///
+    /// - Parameter level: The type of message (`LoggerMessageType`).
+    ///
+    /// - Returns: A Boolean indicating whether a message of the specified type
+    ///           will be logged via the registered `LoggerAPI.Logger`.
+    public class func isLoggingToLoggerAPI(_ level: LoggerMessageType) -> Bool {
```

Comment:
I'm not sure this public API carries its weight. We can always add it in future if needed but for now I think `internal` is fine.